README.md

```diff
@@ -1,7 +1,25 @@
 Awesome Firmware
 ================
 
+## Boot firmware
+
+Code that runs on a processor and helps it boot up.
+
+### First stage
+
+Projects that can be used as the first code in flash.
+
 * [barebox](https://www.barebox.org)
 * [coreboot](https://coreboot.org)
 * [oreboot](https://github.com/oreboot/oreboot)
 * [u-boot](https://www.u-boot.org)
+
+
+### Payload
+
+Code run by the first stage, which can for example offer a wider range of boot options.
+
+* [LinuxBoot](https://www.linuxboot.org/)
+* [petitboot](https://open-power.github.io/petitboot/overview.html)
+* [TianoCore EDK II](https://github.com/tianocore/edk2)
+* [SeaBIOS](https://www.seabios.org/SeaBIOS)
```

Comment:
Should we add u-boot and barebox here as well, since they can be used as second stage bootloader?

Reply:
Good question, duplication has a chance of inconsistencies creeping in later. I'd say yes, though.

Reply:
done